### DISCUSSION TITLE
Remember the last profile selected

### DISCUSSION THREAD

**Discussion by nacesprin** [Natizyskunk/vscode-sftp#238] [Ideas]
I think should be a good feature to be able to remember the last selected profile, instead to must to change the profile each time I run vscode if I don't want to use the defaultProfile.

![image](https://user-images.githubusercontent.com/295375/198243758-9c0545f8-3e49-45f5-995a-14555f28af5a.png)

**Comment by Natizyskunk** [comment]
@nacesprin We'll take a look at it 🙂

**Reply by Natizyskunk** [reply]
@nacesprin Could you open an issue respecting [this](https://github.com/Natizyskunk/vscode-sftp/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=) template please ?
